### ISSUE TITLE
[BUG] Server causes error when loaded with REI

### ISSUE THREAD

**Issue by DMNight6** [GabrielOlvH/Industrial-Revolution#486]
**Description:**
Apparently loading REI with InvRev causes an error for the server.

**Steps to Reproduce:**
Download REI and InvRev and put these files inside the fabric server mods file.

**Technical Information:**
 - Minecraft: 1.19
 - Fabric API: fabric-api-0.58.0+1.19
 - Industrial Revolution: indrev-1.15.0-BETA
 - Modpack: None.

**Logs:**
https://pastebin.com/94Bh5bFn

**Comment by uzervlad** [comment]
Also unable to start a server without REI even though it shouldn't be a required dependency

**Comment by mopsy14** [comment]
Caused by: java.lang.RuntimeException: Rerached side issue when loading REI plugin by Industrial Revolution. Please use "rei_server", "rei_client" or "rei_common" instead.

This is the error causing it, it's most likely because REI did an update.

**Comment by uzervlad** [comment]
No, the issue persists throughout all 1.19 REI versions, so it's likely not the cause. In fact, indrev already uses `rei_common`, although I believe `rei_client` should be enough

**Comment by WbyNghbr** [comment]
@uzervlad Thanks for the comment. That actually works for me as a workaround.

For anyone that may encounter the same problem, here's a quick and dirty solution:

- open the .jar archive (with 7zip.org, for example)
- open `fabric.mod.json` in any editor
- replace `rei_common` with `rei_client`
- save the changes and make sure that .jar archive is updated

After that, you should be able to use the modified .jar archive both on client and server without any issues.

I'm not sure how fabric and rei handle it, but I would imagine that after this change rei_server initialization is no longer triggered, avoiding the problem completely.
